tools/build-scheme-full.R

```diff
@@ -0,0 +1,2 @@
+tinytex::tlmgr_install('scheme-full')
+sys.source('tools/clean-tlpdb.R')
```

Comment:
I believe this is the minimum necessary to create the full bundle right ?

Reply:
Yes, I think so.